lib/util/git/index.ts

```diff
@@ -96,6 +94,14 @@ function checkForPlatformFailure(err: Error): void {
       'GitLab: Commit message does not follow the pattern',
       "Cannot push because commit message does not follow project's push rules",
     ],
+    [
+      ' is not a member of team',
+      '`Restrict commits to existing GitLab users` rule is blocking renovate push. Check renovate `gitAuthor` setting',
```

Comment:
```suggestion
      'The `restrict commits to existing GitLab users` rule is blocking Renovate push. Check the Renovate `gitAuthor` setting',
```

Maybe we should capitalize our branding name. 😉

Reply:
```suggestion
      'The `Restrict commits to existing GitLab users` rule is blocking Renovate push. Check the Renovate `gitAuthor` setting',
```

Uppercase for the rule, as it's named lie that?